### ISSUE TITLE
passing custom CSS

### ISSUE THREAD

**Issue by santhoshmeti** [basvandenberg/ng-select#279]
Is there any options to pass custom class.
like 
<ng-select [options]="inputTemplates"
                            [multiple]="false"
                            formControlName="userName"
                            [placeholder]="'username.create-user.user-placeHolder' | translate"
                            [allowClear]="canClearSelect",
                            **[class]="form-control"**
                        ></ng-select>

**Comment by Areiy** [comment]
Yes, I also couldn't find a way to add custom class. I can wrap ng-select with another div and configure css based on class of this parent div, but it will be useful to have possibility to add class straight to ng-select.

**Comment by Biptor** [comment]
Hi @Areiy 
I need to move the list of options and put them at the right of the input field. This is possible wrapping the ng-select with another div?

**Comment by Areiy** [comment]
Hi @Biptor
Try to wrap your ng-select with div:
```
<div class="ng-select-right">
    <ng-select 
          [options]="myOptions"
          [(ngModel)]="mySelect">
    </ng-select>
</div>
```

and add styles:

```
.ng-select-right select-dropdown{
    position: absolute;
    top: -100%;
    right: 0;
}
```
